server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -226,6 +228,8 @@ export function configureWebSocketServices(
         // Timer to check token expiry for this socket connection
         let expirationTimer: NodeJS.Timer | undefined;
 
+        const connectionCountLogger = new ConnectionCountLogger(process.env.NODE_NAME, cache);
```

Comment:
Should we configure NODE_NAME here in OSS alfred-deployment.yaml as an example of how to set this env variable?

Reply:
I can. I thought since we only need it in FRS, we don't really need to add it here. Do you still want me to?

Reply:
added

Reply:
I think that is good practice to do that. Since the change is being added here in GitHub, it might be confusing for other consumers of these packages to understand how that is set if there is no example.